src/components/app/ProfileViewer.js

```diff
@@ -42,11 +42,11 @@ type Props = {
   changeTabOrder: (number[]) => Action,
 };
 
-class ProfileViewer extends PureComponent {
-  props: Props;
-  state: {
-    isMounted: boolean,
-  };
+type State = {
+  isMounted: boolean,
+};
+
+class ProfileViewer extends PureComponent<Props, State> {
```

Comment:
This seems inconsistent in how we are defining our `Props`. It appears you've removed the `props: Props` statement, but left it other places.

Reply:
This should be the one only way to define props now, I should have removed all of them now (I just forgot one as you properly found above :) ). Have you found other places ?